package.json

```diff
@@ -36,6 +37,10 @@
   },
   "devDependencies": {
     "eslint": "^4.8.0",
+    "gulp": "^3.9.1",
+    "gulp-jasmine-browser": "^1.7.1",
```

Comment:
Is gulp being used? It doesn't seem like this should be added to package.json

Reply:
Yeah not using gulp, so please remove this